src/__tests__/store/reducers/apiResponse.reducer.test.js

```diff
@@ -98,6 +101,7 @@ describe('apiResponseReducer', () => {
                 expect(result.confirmationModal.api).toEqual(api);
                 expect(result.confirmationModal.apiName).toEqual(apiName);
                 expect(result.confirmationModal.message).toEqual(message);
+                expect(result.confirmationModal.shouldRedirect).toEqual(shouldRedirect);
```

Comment:
also I think if you need to check every value of the confirmation modal maybe use `toBe`, in the other case maybe just check the whole object.